neptune/projects.py

```diff
@@ -297,11 +296,11 @@ def exception_handler(exc_type, exc_val, exc_tb):
                 websocket_factory=websocket_factory, abort_impl=abort_impl, experiment_id=experiment.internal_id)
             experiment._aborting_thread.start()
 
-        if upload_stdout:
+        if upload_stdout and not is_notebook():
```

Comment:
Since is_notebook handled by exception, it would be best if we didn't call it multiple times